marklogic-data-hub-central/ui/src/config/tooltips.config.js

```diff
@@ -111,7 +111,7 @@ const ModelingTooltips = {
   pii: 'Personally identifiable information, or data this is sensitive in that it can identify a particular person and can be used fraudulent purposes.',
   sort: 'Enables display in ascending or descending order.',
   facet: 'Use when searching, for dividing values of properties into counts, aggregating, grouping, and more.',
-  advancedSearch: 'Enables wildcard search, search suggestions, and type-ahead.',
+  wildcard: 'Enables wildcard search.',
```

Comment:
Curation components have RTL tests for tooltip. Might be good to have them for Modeling as well. Will leave it upto you.

Reply:
I can look into adding them